demos/python_demos/object_detection_demo/models/pipelines.py

```diff
@@ -0,0 +1,88 @@
+import threading
```

Comment:
It's strange to have pipeline in models folder.

Reply:
Done